Parsers/VSISDMI.cs

```diff
@@ -1724,296 +1728,1124 @@ public VSISDMI()
+                    Length = 8,
+                    Comment = "Spare for alignment"
+                },
+                new BitField
+                {
+                    Name = "MMI_STM_L_TEXT_IPT",
+                    BitFieldType = BitFieldType.UInt16,
+                    Length = 16,
+                    Comment =
+                        "Number of characters in text message (range: 0-40)." +
+                        "\r\nNote: If variable = 0 the MMI will remove the STM test request message."
+                },
+                new BitField
+                {
+                    Name = "MMI_STM_X_TEXT",
+                    BitFieldType = BitFieldType.HexString,
+                    Comment = "Text character",
+                    VariableLengthSettings = new VariableLengthSettings
+                    {
+                        Name = "MMI_STM_L_TEXT_IPT",
+                        ScalingFactor = 8
+                    }
+                }
+            }
+        };
+
+        // checked 20190510 2.11
+        public static DataSetDefinition EVC_28 => new DataSetDefinition
+        {
+            Name = "EVC_28 MMI_ECHOED_SET_VBC_DATA",
+            Comment =
+                "This packet shall be sent from ETC to MMI when the driver has finished the 'Set VBC' data entry by pressing the \"Yes\" button and all checks have passed. " +
+                "The packet starts the 'Set VBC' Data Validation window / procedure at MMI. ",
+            Identifiers = new List<string>
+            {
+                "230530230",
+                "230535230"
+            },
+            BitFields = new List<BitField>
+            {
+                MMI_M_PACKET,
+                MMI_L_PACKET,
+                new BitField
+                {
+                    // TODO How do you set just this BitField with the InvertBits parameter?
+                    Name = "MMI_M_VBC_CODE_",
+                    BitFieldType = BitFieldType.UInt32,
+                    Length = 32,
+                    Comment = "VBC Identifier, bit inverted"
+                }
+            }
+        };
+
+        // checked 20190510 2.11
+        public static DataSetDefinition EVC_29 => new DataSetDefinition
+        {
+            Name = "EVC_29 MMI_ECHOED_REMOVE_VBC_DATA",
+            Comment =
+                "This packet shall be sent from ETC to MMI when the driver has finished the 'Remove VBC' data entry by pressing the \"Yes\" button and all checks have passed. " +
+                "The packet starts the 'Remove VBC' Data Validation window / procedure at MMI. ",
+            Identifiers = new List<string>
+            {
+                "230530240",
+                "230535240"
+            },
+            BitFields = new List<BitField>
+            {
+                MMI_M_PACKET,
+                MMI_L_PACKET,
+                new BitField
+                {
+                    // TODO How do you set just this BitField with the InvertBits parameter?
```

Comment:
make a nested dataset and set InvertBits on it

Reply:
Nested DataSet implemented